spanner/spanner_samples.rb

```diff
@@ -1223,6 +1234,167 @@ def query_with_timestamp project_id:, instance_id:, database_id:
+    row["Heights"].each { |height| puts height.nil? }
+  end
+
+  # [END spanner_write_read_null_int64_array]
+end
+
+def write_read_int64_array project_id:, instance_id:, database_id:
+  # [START spanner_write_read_int64_array]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+  require "securerandom"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+
+  box_id = SecureRandom.uuid
+  client.insert "Boxes", BoxId: box_id, Heights: [10, 11, 12]
+  results = client.read "Boxes", [:BoxId, :Heights], keys: box_id
+
+  results.rows.each do |row|
+    puts row["Heights"]
+  end
+
+  # [END spanner_write_read_int64_array]
+end
+
+def write_read_empty_float64_array project_id:, instance_id:, database_id:
+  # [START spanner_write_read_empty_float64_array]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+  require "securerandom"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+
+  box_id = SecureRandom.uuid
+  client.insert "Boxes", BoxId: box_id, Weights: []
+  results = client.read "Boxes", [:BoxId, :Weights], keys: box_id
+
+  results.rows.each do |row|
+    puts row["Weights"].empty?
+  end
+
+  # [END spanner_write_read_empty_float64_array]
+end
+
+def write_read_null_float64_array project_id:, instance_id:, database_id:
+  # [START spanner_write_read_null_float64_array]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+  require "securerandom"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+
+  box_id = SecureRandom.uuid
+  client.insert "Boxes", BoxId: box_id, Weights: [nil, nil, nil]
+  results = client.read "Boxes", [:BoxId, :Weights], keys: box_id
+
+  results.rows.each do |row|
+    row["Weights"].each { |weight| puts weight.nil? }
+  end
+
+  # [END spanner_write_read_null_float64_array]
+end
+
+def write_read_float64_array project_id:, instance_id:, database_id:
+  # [START spanner_write_read_float64_array]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+  require "securerandom"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+
+  box_id = SecureRandom.uuid
+  client.insert "Boxes", BoxId: box_id, Weights: [10.001, 11.1212, 104.4123101]
+  results = client.read "Boxes", [:BoxId, :Weights], keys: box_id
+
+  results.rows.each do |row|
+    puts row["Weights"]
+  end
+
```

Comment:
```suggestion
```

Reply:
@skuruppu Removed new line from all the occurrences as mentioned above.